index.html

```diff
@@ -6,10 +6,14 @@
     <title>HTML & CSS Training</title>
     <meta name="description" content="training">
     <meta name="author" content="Wolox">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
     <link rel="stylesheet"  type="text/css" href="application.css">
   </head>
   <body>
     <div class="container">
+      <div class="back-arrow">
```

Comment:
In order to reduce the elements on the DOM have you tried to remove this div? I believe you can achieve the same results